ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator/gcs.rs

```diff
@@ -137,7 +137,11 @@ impl FileStoreOperator for GcsFileStoreOperator {
         chain_id: u64,
         version: u64,
     ) -> anyhow::Result<()> {
-        let metadata = FileStoreMetadata::new(chain_id, version);
+        let metadata = FileStoreMetadata::new(
+            chain_id,
+            version,
+            StorageFormat::JsonBase64UncompressedProto,
```

Comment:
might not want to hardcode this.

Reply:
hmm does FileStoreMetadata even need to specify a storage format? it should be unaffected by this compression change. if we don't need storage format, we should remove it as a var.

Reply:
added the check; this prevents people trying to mix compression with uncompressed stuff.